src/views/waterTemperature/WaterTemperature.vue

```diff
@@ -322,12 +320,14 @@
                 img.src = src;
                 img.onload = () => resolve(img);
                 img.onerror = reject;
-                setTimeout(function(){reject('timeout')}, 20000)
+                setTimeout(function(){reject('timeout')}, 60000)
               }).then(function(img){
+                console.log(img)
```

Comment:
heh, remove next time you're in here if you keep this

Reply:
Sigh good catch

Reply:
Well I want the console log if there is an error.  So gonna keep that one in